vscode-extension/editor/src/utils/vscodeUtils.ts

```diff
@@ -10,7 +9,7 @@ import { JSONValue } from "aiconfig";
  * For more information, see https://code.visualstudio.com/api/extension-guides/webview#getstate-and-setstate
  */
 export type WebviewState = {
-  aiconfigState?: ClientAIConfig;
+  aiconfigState?: AIConfig;
```

Comment:
This is the proper type since this comes directly from the aiconfig server. ClientAIConfig is exclusive to the editor component since it contains _ui fields